odoo/addons/base/models/ir_cron.py

```diff
@@ -256,7 +256,15 @@ def _acquire_one_job(cls, cr, job_ids):
               AND id in %s
             ORDER BY priority
             LIMIT 1 FOR NO KEY UPDATE SKIP LOCKED
-        """, [job_ids])
+        """
+        try:
+            cr.execute(query, [job_ids], log_exceptions=False)
+        except psycopg2.extensions.TransactionRollbackError:
+            # don't log serialization errors
```

Comment:
That's not a useful comment, the code tells us why *that* we're not logging serialization errors, what'd be useful is knowing *why*.